lib/kuzzle/kuzzle.js

```diff
@@ -187,11 +191,17 @@ class Kuzzle extends KuzzleEventEmitter {
 
       // ACLs must be loaded after plugins has started, otherwise importing
       // users with credentials will fail and will prevent Kuzzle from starting
-      if (options.securities) {
-        this.log.info('[ℹ] Loading default rights... This can take some time.');
-        await this.ask('core:security:load', options.securities, { force: true });
-        this.log.info('[✔] Default rights loaded');
-      }
+      this.log.info('[ℹ] Loading default rights... This can take some time.');
+
+      await this.ask('core:security:load', options.securities, { force: true });
```

Comment:
Users, profiles and rights should be loaded either with the deprecated `securities` object or with the new `import.*` objects.

If someone try to use both then we should stop the start sequence